cmd/admin-bucket-handlers.go

```diff
@@ -205,6 +205,11 @@ func (a adminAPIHandlers) SetRemoteTargetHandler(w http.ResponseWriter, r *http.
 		}
 		target = tgt
 	}
+
+	if target.BandwidthLimit*8 < 100*1000*1000 {
```

Comment:
```suggestion
	if target.BandwidthLimit*8 < 100*humanize.Mbyte {
```

Reply:
and also why is it multiplied by 8?

Reply:
bandwidth is taken as input in bits/sec in mc but sent to server as bytes.

Reply:
Yeah a comment will be important here @poornas

Reply:
@poornas btw client allows setting bandwidth multiple types

>   --bandwidth value             Set bandwidth limit in bits per second (K,B,G,T for metric and Ki,Bi,Gi,Ti for IEC units)

It in-fact sends to server in bytes

```
func getBandwidthInBytes(bandwidthStr string) (bandwidth uint64, err error) {
        if bandwidthStr != "" {
                bandwidth, err = humanize.ParseBytes(bandwidthStr)
                if err != nil {
                        return
                }
        }
        bandwidth = bandwidth / 8
        return
}
```

This doesn't look like correct change